lib/json_spec/indifference.rb

```diff
@@ -0,0 +1,25 @@
+module JsonSpec
+  module Indifference
+    extend self
+
+    def toggle_indifference( toggle )
+      @indifferent = !!toggle
+    end
+
+    def indifferize(ruby)
+      return ruby unless @indifferent
+
+      case ruby
+      when Hash
+        ruby.sort.inject({}) do |hash, (key, value)|
```

Comment:
Were hashes ever failing due to order dependence issues? If not, we should not bother sorting the incoming hash.